### DISCUSSION TITLE
Archer Sniper Rifle (MISHIMA) [U]

### DISCUSSION THREAD

**Discussion by 21-ric** [secretcowlevel/DoomtrooperPublic#51] [Card Balance]
Archer Sniper Rifle (MISHIMA) [U]
EQUIPMENT, FIREARM, FIRSTSTRIKE
[S] +2. 1 Action: +1 to [S] (until end of turn)

My suggestion :
EQUIPMENT, FIREARM, FIRSTSTRIKE
[S] +2. 1 Action: +3 to [S] (until end of turn)

If you spend 2 actions you will have a FIREARM with +8 to [S] and FIRSTRIKE.

**Comment by Michalx11** [comment]
I have to disagree with this suggestion. That's way too much. I think is perfect as it is

**Comment by Sol-au-fein** [comment]
You can spend one action, and only one, to get additional +2S or perhaps +3S. 
Dont forget it has also Firststrike, so in hands of Slayer Hatamoto it could be bit strong.
